### PR TITLE
Fix - Form Setup yesno

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "FIX", "CLOSE" or "NEW" section* (use uppercase to have the PR appears into the ChangeLog, lowercase will not appears)
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed texts with meaningful information*

# FIX|Fix #[*issue_number Short description*]
[*Long description*]

# CLOSE|Close #[*issue_number Short description*]
[*Long description*]

# NEW|New [*Short description*]
[*Long description*]

### CODE REVIEW

File: htdocs/core/class/html.formsetup.class.php [hunk 1]
```diff
@@ -835,7 +835,11 @@ public function generateInputField()
 		} elseif ($this->type== 'color') {
 			$out.=  $this->generateInputFieldColor();
 		} elseif ($this->type == 'yesno') {
-			$out.= $this->form->selectyesno($this->confKey, $this->fieldValue, 1);
+			if (!empty($conf->use_javascript_ajax)) {
```
Comment (eldy) [inline htdocs/core/class/html.formsetup.class.php:838]: Do you have an example of form popup and what is result (can you provide a screenshot) ?

Comment (bb2a) [reply htdocs/core/class/html.formsetup.class.php:838]: @eldy it's not in popups it's in modules setup pages

Comment (eldy) [reply htdocs/core/class/html.formsetup.class.php:838]: Sorry, can you provide an example of screen (screenshot) where we can test this ?

Comment (bb2a) [reply htdocs/core/class/html.formsetup.class.php:838]: Befor
image

After
image

**Pull request merged**